### DISCUSSION TITLE
A quick launch summary on server startup.

### DISCUSSION THREAD

**Discussion by VonSparq** [PurpurMC/Purpur#720] [__Dismissed__] [answered]
Hi!
Would it be possible for you to add a short server startup summary just after the server starts up and prints out Done?
I was thinking of something like this:
"The server has been started up successfully with **X** errors and **Y** warnings. **A** _out of B_ Plugins and **C** _out of D_ Datapacks have been loaded."
I think it'd be very helpful for quickly checking up if there's anything you need to check in the startup log.

**Comment by YouHaveTrouble** [comment]
Maybe only make it so it displays when debug is enabled in server.properties?

**Reply by VonSparq** [reply]
I'd like for it to be it it's own option in purpur.yml, tieing it to debug will mean you have to have debug enabled all the time to see the summary, which will result in a big increase in console output I think.

**Comment by encode42** [comment, marked as answer]
I feel this would be redundant. If issues occur during startup, then a giant stack trace, or at least a `[WARN]`/`[ERROR]` will appear. Easy to `CTRL` + `F` for these keywords to "tally" the amount of issues.

`/plugins` also highlights plugins that have encountered issues while enabling. *I believe* the datapack list command does so as well. 

Issues also occur once. Once you fix them, this feature would never be used again. Unless big changes are made, even then, the above still applies.

**Reply by VonSparq** [reply]
Sure it can be done if you pay attention to the console as the server boots up or if you go into the log and check.
I still think this would be a valuable feature, giving you a quick at-a-glance check if everything went smoothly after the server is finished starting up.

**Reply by katekerllenevich** [reply]
This really isn't Purpurs place, and we have no plans to include this. You could make a simple easy plugin to do it in about 15 minutes.

**Reply by VonSparq** [reply]
Fair enough, though I find the suggestion that "I" would be able to do this quite amusing :D
Still, it's understandable, thank you for your consideration and have a great day!
